modules/running-cluster-oci-agent-based.adoc

```diff
@@ -4,39 +4,30 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="running-cluster-oci-agent-based_{context}"]
-= Running a cluster on OCI
+= Running a cluster on {oci}
 
 To run a cluster on {oci-first}, you must upload the generated agent ISO image to the default Object Storage bucket on {oci}. Additionally, you must create a compute instance from the supplied base image, so that your {product-title} and {oci} can communicate with each other for the purposes of running the cluster on {oci}.
 
+[NOTE]
+====
+{oci} supports the following {product-title} cluster topologies:
+
+* Installing an {product-title} cluster on a single node.
+* A highly-availabile cluster that has a minimum of three control plane instances and two compute instances.
+* A compact three-node cluster that has a minimum of three control plane instances.
+====
+
 .Prerequisites
 
 * You generated an agent ISO image. See the "Creating configuration files for installing a cluster on OCI" section.
 
 .Procedure
 
-. Upload the agent ISO image to Oracle’s default Object Storage bucket and then import the agent ISO image as a custom image to this bucket. You must then configure the custom image to boot in Unified Extensible Firmware Interface (UEFI) mode. See link:https://docs.oracle.com/en-us/iaas/secure-desktops/create-custom-image.htm[Creating a custom image] and link:https://docs.oracle.com/en-us/iaas/Content/Compute/Tasks/configuringimagecapabilities.htm#ariaid-title5[Using the Console] in Oracle’s documentation.
-+
-For example, from *Compute* -> *Custom images*, import the agent ISO image to the bucket, and enter values in the following fields:
-+
-* Name: *oci-cluster*
-* Bucket: Select the bucket that contains the agent ISO image
-* Object name: Select the name of the agent ISO
-* Image type: *QCOW2*
-
-. After the image imports, go to the *Edit image capabilities* setting and ensure that only `UEFI_64` is selected for the *Firmware* field.
+. Upload the agent ISO image to Oracle’s default Object Storage bucket and import the agent ISO image as a custom image to this bucket. Ensure you that you configure the custom image to boot in Unified Extensible Firmware Interface (UEFI) mode. For more information, see link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-agent-image-creation.htm[Creating the {product-title} ISO Image (Oracle documentation)].
```

Comment:
```suggestion
. Upload the agent ISO image to Oracle’s default Object Storage bucket and import the agent ISO image as a custom image to this bucket. Ensure you that you configure the custom image to boot in Unified Extensible Firmware Interface (UEFI) mode. For more information, see link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-agent-image-creation.htm[Creating the {product-title} Discovery ISO Image (Oracle documentation)].
```

Reply:
The "discovery"  ISO image is affiliatd with the Assisted Installer and not the Agent-based Installer, so I'll drop this update.

Reply:
gotcha, sounds good 👍